tests/package.json

```diff
@@ -35,7 +35,7 @@
     "watch": "npm-watch",
     "build": "cargo build --release",
     "non-ci-test": "mocha -r ts-node/register 'non_ci_tests/**/test-*.ts'",
-    "test-single": "mocha -r ts-node/register 'tests/test-balance.ts'"
+    "test-single": "mocha -r ts-node/register 'tests/test-crowdloan.ts'"
```

Comment:
Is this just something used for convenience to test whatever single test we want to run? Not a big deal, but this might be a nuisance for merging...

Reply:
yes it is just for convinience. why could it be a nuisance?

Reply:
Because it also got modified in https://github.com/PureStake/moonbeam/pull/445 for example. So there could be merge conflicts between the two.